ironfish/src/rpc/routes/wallet/getNotes.ts

```diff
@@ -48,8 +50,13 @@ router.register<typeof GetAccountNotesStreamRequestSchema, GetAccountNotesStream
       const transaction = await account.getTransaction(transactionHash)
       Assert.isNotUndefined(transaction)
 
+      const asset = await node.chain.assets.get(note.assetIdentifier())  
+
+      Assert.isNotUndefined(asset)
+
       request.stream({
         amount: note.value().toString(),
+        assetIdentifier: asset.name.toString('hex') || "$IRON",
```

Comment:
This is a bug your passing the name as an identifier and hexing it.

Reply:
+1